### ISSUE TITLE
Some confusion about the trainning time. 

### ISSUE THREAD

**Issue by LemonMi** [yzhou359/3DIndoor-SceneGraphNet#2]
Hello,
We were trying to train the network on your original code, but we have some trouble that the training time is so long. It takes about 2 hours per epoch using bedroom dataset on a GeForce 1080Ti GPU or a GeForce 2060 GPU. 
I'm wondering if you could tell me how many GPU you used  for the training? And what about the batchsize and the learning rate?

**Comment by yzhou359** [comment]
Hi,
We had a similar training time as you did - 1-2 hours per each epoch with a single 1080ti GPU. For bedroom, it takes around 20 epochs to converge. That's indeed an unsolved issue with the current code implemented with Torchfold. We used batch_size=2, learning rate=5e-4.
